language-design.md

```diff
@@ -17,7 +17,41 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+Quote 1: "Why shouldn’t those 'interfaces' be as humanely designed as the ones we tap and swipe?" [Pavlus, 2012].
+
+Here Pavlus asks why programming languages' interfaces are not as "humanely
+designed" as software's graphical interfaces are. This quote stood out to us
+because Pavlus is trying to make a point, but actually ends up asking a
+question that is very easy to answer! One answer is that in general, software
+(programming languages included) is designed towards its users. The software
+Pavlus is referring to is probably targeted at a much broader audience, so it
+needs to be easier. Also, we take issue with Pavlus implying that programming
```

Comment:
I think that this article was written by someone who didn't really know what they were getting into...he raises a question that is meant to open a discussion but his question is the wrong one. What he means to be saying (i think) is: "is it impossible to design a programming language that is both intuitive AND functional to the necessary degree?"

I think that that question is a lot more interesting and I tend to thing that the answer (so far, at least) has been "yes" but it won't always have to be that way (maybe)